projects/bind9/Dockerfile

```diff
@@ -0,0 +1,34 @@
+# Copyright 2016 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+################################################################################
+
+FROM gcr.io/oss-fuzz-base/base-builder
+ENV DEBIAN_FRONTEND noninteractive
+RUN apt-get -y update && apt-get -y dist-upgrade
+RUN apt-get -y install		\
+	autoconf		\
+	automake		\
+	autotools-dev		\
+	bison			\
+	build-essential		\
+	libtool			\
+	libtool-bin		\
+	libssl-dev		\
+	libuv1-dev		\
+	pkg-config		\
+	zip
+RUN git clone --depth 1 --branch ondrej/oss-fuzz https://gitlab.isc.org/isc-projects/bind9.git
```

Comment:
can we use the master branch? that's the recommended approach, as it allows to find the most recent regressions and verify bug fixes as soon as possible

Reply:
Yes, but currently it needs few extra bits that haven’t been merged to main yet. I’ll also add the support for oss-fuzz to autoconf as it will make the integration with native build easier.

Reply:
Thanks for clarifying. Sounds good for now, but let's try to switch it to the master in future (when possible).